src/Channel.ts

```diff
@@ -4,7 +4,13 @@ export type OnMessageCallback = (message: {}) => void
 
 export interface Channel {
     timeout: number
-    autoReconnect: () => void
+    /**
+     * Orders the channel to reconnect.
+     *
+     * @remarks To implement in order to beneficiate from the auto-reconnect feature.
```

Comment:
Typo with benefit instead of beneficiate

Reply:
fixed